### PR TITLE
MTM-44339: oAuth internal for microservices which does not uses c8y SDKs

### PR DESCRIPTION
**Pull request by lukaszsoszynskisag**

### CODE REVIEW

**Review by 509132 — APPROVED** [review]

**Review by 509132 — APPROVED** [review]

File: content/microservice-sdk/concept-bundle/security.md [hunk 1]
```diff
@@ -13,13 +13,87 @@ Microservices typically provide a REST API and {{< product-c8y-iot >}} provides
 
 ### Authentication and authorization
 
-A request to a microservice can be authenticated using basic authentication or OAuth. In case of basic authentication the flow is fairly simple, as credentials can be read and utilized for further authentication to the platform.
+A request to a microservice can be authenticated using various authentication mechanisms supported by {{< product-c8y-iot >}} like:
+1. Basic authentication
+2. OAI-Secured
```
Comment (ArturBator) [inline content/microservice-sdk/concept-bundle/security.md:18]: `OAI-Secured`, or `OAI-Secure`?

Comment (BeateRixen) [reply content/microservice-sdk/concept-bundle/security.md]: correct, fixed

File: content/microservice-sdk/concept-bundle/security.md [hunk 2]
```diff
@@ -13,13 +13,87 @@ Microservices typically provide a REST API and {{< product-c8y-iot >}} provides
...
+3. SSO
+4. JWT token authentication (depricated)
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/microservice-sdk/concept-bundle/security.md]: fixed

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@lukaszsoszynskisag From my perspective this can be merged. Or do you need another approval from someone particular? Should this go into 10.13 as well?

**Review by MWindel — APPROVED** [review]

**Comment by lukaszsoszynskisag** [comment]
@BeateRixen This change should be included in 10.13 as well

**Comment by BeateRixen** [comment]
> @BeateRixen This change should be included in 10.13 as well

Done.

**Pull request merged**